src/ServicePulse.Host/vue/src/components/monitoring/EndpointListRow.vue

```diff
@@ -136,7 +136,7 @@ function formatGraphDecimal(input, deci) {
         {{ endpoint.isStale == true || endpoint.isScMonitoringDisconnected == true ? "" : formatGraphDuration(endpoint.metrics.processingTime).value }}
         <strong v-if="endpoint.isStale && !endpoint.isScMonitoringDisconnected" v-tooltip title="No metrics received or endpoint is not configured to send metrics">?</strong>
         <strong v-if="endpoint.isScMonitoringDisconnected" v-tooltip title="Unable to connect to monitoring server">?</strong>
-        <span v-if="endpoint.isStale == false && endpoint.isScMonitoringDisconnected == false"> {{ formatGraphDuration(endpoint.metrics.processingTime).unit }}</span>
+        <span v-if="!endpoint.isStale && !endpoint.isScMonitoringDisconnected"> {{ formatGraphDuration(endpoint.metrics.processingTime).unit }}</span>
```

Comment:
This type of pattern is used everywhere. Should we extract this as a computed `showData` or something? Maybe not in this PR.

Reply:
can discuss further, definitely not as part of this PR (i just had to change this because it wasn't working)